### PR TITLE
fix: modal bg

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]

File: packages/shared/src/components/post/PostContentContainer.tsx [hunk 1]
```diff
@@ -7,7 +7,10 @@ interface PostContentContainerProps {
   children: ReactNode;
 }
 
-const BodyContainer = classed('div', 'flex flex-col w-full pb-6');
+const BodyContainer = classed(
+  'div',
+  'flex flex-col w-full pb-6 bg-background-default',
+);
```
Comment (capJavert) [inline packages/shared/src/components/post/PostContentContainer.tsx:13]: How did this break just now? Seems global.

Comment (sshanzel) [reply packages/shared/src/components/post/PostContentContainer.tsx:13]: Yeah, actually quite not sure 😅

Comment (sshanzel) [reply packages/shared/src/components/post/PostContentContainer.tsx:13]: I will investigate after confirming the fix on prod.

**Review by nensidosari — APPROVED** [review]

File: packages/shared/src/components/post/PostContentContainer.tsx (hunk 1 above)
Comment (capJavert) [reply packages/shared/src/components/post/PostContentContainer.tsx:13]: fylla!

**Pull request merged**